spec/fixtures/responses/mutations/freak_create/freak_create.json

```diff
@@ -1,16 +1,11 @@
 {
   "status": 200,
   "body": {
-    "data": {
-      "freakCreate": {
-        "firstName": "john",
-        "lastName": "doe",
-        "description": "ceva",
-        "email": "freak@gmail.com",
-        "norm": {
-          "name": "full_time"
-        }
+    "data": null,
+    "errors": [
+      {
+        "message": "Cannot return null for non-nullable field Freak.role"
```

Comment:
That's not what we expect. We expect the data from the `Freak` to be returned.

Reply:
Ok, I fix this